### PR TITLE
added Mean to raw_ops

### PR DESCRIPTION
**Pull request by dovaaa**
- [ ] #5112

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -1432,3 +1432,46 @@ def test_tensorflow_GreaterEqual(dtype_and_x, as_variable, fw, native_array):
...
+    keep_dims,
+):
+    dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:1469]: we are not passing `fw` in the `test_frontend_function` now, this needs to be removed only from here.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey! 
First of all thank you soo much for contributing to ivy! 🙂
Your PR looks amazing ✨!
Just requested a few change and left a comment on it which needs to be addressed before we can merge your PR! 🙂
Thanks!

**Comment by zaeemansari70** [comment]
Hey @dovaaa 🙂, I will link this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained [here](https://lets-unify.ai/ivy/contributing/1_the_basics.html#todo-list-issues
Thanks!

**Comment by dovaaa** [comment]
All right, Will Do!

**Pull request merged**